components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -480,6 +501,17 @@ - (void)viewWillLayoutSubviews {
   [super viewWillLayoutSubviews];
 
   [self setupLayout];
+  UIView *topView;
+  if (self.headerViewController) {
+    topView = self.headerViewController.view;
+  } else {
+    topView = self.contentViewController.view;
+  }
+  self.shadowedView.frame = topView.frame;
+  if (topView.layer.mask) {
+    self.shadowedView.layer.shadowPath = ((CAShapeLayer *)topView.layer.mask).path;
```

Comment:
Shouldn't there be a class check here before casting?

It also seems that because `mask` is defined as `__kindof CALayer *`, a cast shouldn't be necessary. Have I missed something?

This code generates no warnings or errors for me because of the type definition of `mask`:

```objc
  UIView *view = [[UIView alloc] init];
  CAShapeLayer *shapeLayer = view.layer.mask;
  (void)shapeLayer;
```

Reply:
Done.